tmt/steps/report/junit.py

```diff
@@ -359,19 +357,19 @@ def _read_log_filter(log: Path) -> str:
         except etree.XMLSyntaxError as error:
             phase.verbose('rendered XML', xml_data, 'red')
             raise tmt.utils.ReportError(
-                'The generated XML output is not a valid XML file. Use `--verbose` argument '
-                'to show the output.') from error
+                'The generated XML output is not a valid XML file (SyntaxError). Use `--verbose` '
+                'argument to show the output.') from error
 
-    # Do not be fooled by the `encoding` parameter: even with `utf-8`,
-    # `tostring()` will still return bytes. `unicode`, on the other
-    # hand, would give us a string, but then, no XML declaration.
-    # So, we get bytes, and we need to apply `decode()` on our own.
+    # Do not be fooled by the `encoding` parameter: even with `utf-8`, `tostring()` will still
+    # return bytes. `unicode`, on the other hand, would give us a string, but then, no XML
+    # declaration.  So, we get bytes, and we need to apply `decode()` on our own.
     xml_output: bytes = etree.tostring(
         tree_root,
         xml_declaration=True,
         pretty_print=prettify,
-        # The 'utf-8' encoding must be used instead of 'unicode', otherwise
-        # the XML declaration is not included in the output.
+
+        # The 'utf-8' encoding must be used instead of 'unicode', otherwise the XML declaration is
+        # not included in the output.
```

Comment:
I see several changes to comments, just reformatting, no actual modifications of their text. Is that on purpose? The patch could be a bit smaller. Here, then lines 363 and 337.

Reply:
Sure. One of the comments was actually changed - I added "SyntaxError" to the exception message to be more explicit. But right now I think this is unnecessary because we are raising `ReportError` from `XMLSyntaxError`. 

The last comment modification with `S320` message - I just think it's better to put it on one line. It's a very small change, but I can also revert it if you want.

I've modified my changes, thanks.